tests/storage/localfssd_test.py

```diff
@@ -864,7 +864,9 @@ def test_dump_sd_volumes(monkeypatch, tmp_repo, user_mount, user_domain):
             volUUID=vol_uuid,
             desc="test",
             srcImgUUID=sc.BLANK_UUID,
-            srcVolUUID=sc.BLANK_UUID)
+            srcVolUUID=sc.BLANK_UUID,
+            sequence=sc.DEFAULT_SEQUENCE,
```

Comment:
Why test this with default value? this should be tested with real value like the block storage tests.

Reply:
changed to `expected_sequence`